frontend/js/src/utils/APIService.ts

```diff
@@ -1635,4 +1661,27 @@ export default class APIService {
     await this.checkStatus(response);
     return response.json();
   };
+
+  searchPlaylistForUser = async (
+    searchQuery: string,
+    musicbrainzID: string,
+    count: number = 25,
+    offset: number = 0
+  ): Promise<any> => {
```

Comment:
This method anbd the one below should be typed more accurately.
I think you've created the right type for it:
```suggestion
  ): Promise<PlaylistTypeSearchResult> => {
```